src/components/List/components/ListItem.tsx

```diff
@@ -74,6 +88,12 @@ export class ListItem<T = unknown> extends React.Component<ListItemProps<T>> {
         });
     };
 
+    private onKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
+        if (this.props.onClick && [' ', 'Spacebar'].includes(event.key)) {
```

Comment:
These are breaking changes. Users handle this actions by themselves. And now for example `Select` fires double select action in case of using keyboard